### PR TITLE
Upgrades insert_by_period materialization to dbt 0.13.0

### LINKED ISSUE #127: insert_by_period_materialization fails with dbt 0.13.0
Using dbt=0.13.0, the `insert_by_period` materialization fails with error

```
Compilation Error in macro materialization_insert_by_period_default (macros/materializations/insert_by_period_materialization.sql)
  get_relation() missing 1 required positional argument: 'database'
```

Looks like it's line 60 in the macro code:

```
{%- set old_relation = adapter.get_relation(schema=schema, identifier=identifier) -%}
```

### PR DESCRIPTION
**Pull request by clausherther**
Fixes incompatibilities in `insert_by_period` materialization with 0.13.0 adapter calls and updates repo to require dbt version 0.13.0.

Fixes #127

### CODE REVIEW

File: dbt_project.yml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 name: 'dbt_utils'
 version: '0.1.0'
 
+require-dbt-version: "0.13.0"
```
Comment (drewbanin) [inline dbt_project.yml]: We can set this to a version range, such that dbt-utils will work with future versions of dbt!

```suggestion
require-dbt-version: ">=0.13.0"
```

Comment (clausherther) [reply dbt_project.yml]: Makes sense. Done!

**Comment by drewbanin** [comment]
@clausherther running the tests now - will merge this when they pass :)

**Comment by clausherther** [comment]
Thanks!!

**Comment by drewbanin** [comment]
Thank _you_! I'm going to tackle https://github.com/fishtown-analytics/dbt-utils/issues/129, then cut a new release of this package

**Pull request merged**